doc/changelogs/CHANGELOG_V20.md

```diff
@@ -39,6 +39,191 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="20.4.0"></a>
```

Comment:
Anchor is missing in the table above.
Refs: https://github.com/nodejs/node-core-utils/issues/686